src/openrct2/peep/Staff.h

```diff
@@ -86,6 +86,8 @@ void game_command_set_staff_name(
 void game_command_pickup_staff(
     int32_t* eax, int32_t* ebx, int32_t* ecx, int32_t* edx, int32_t* esi, int32_t* edi, int32_t* ebp);
 
+rct_peep* staff_get_peep_from_sprite_id(uint16_t sprite_id);
```

Comment:
This name is a bit confusing. What about simply `get_staff`? The argument name already makes it clear it's from a sprite index.

Reply:
Id say fetching a object pointer from a sprite id feels like a specific case and does not deserve a generic simple name like get_staff. If the SpriteID was a typechecked variable id agree though.
In my opinion on the callsite:
rct_peep* peep = staff_get_peep_from_sprite_id(w->number);
is more clear whats going on than:
rct_peep* peep = get_staff(w->number);

I would name it "get_staff_from_sprite_id" but since every function started with "staff_" i figured i continue with that standard.

Reply:
I disagree, it doesn't feel like a very specific case to me, the function is quite generic. I do agree with the callsite looking confusing - the same thing is done for ride windows (`Ride* ride = get_ride(w->number);`) - but still acceptable in my opinion.

If you want to make it more clear, you can always store the window number in a nicer named temporary variable.

Peep code has been refactored not too long ago, most of the `staff_*` functions are leftover C functions that haven't been refactored in a nice way yet. I'm not sure if this is really the right place for this function, but it's fine for now. This needs to be refactored further later on.

Reply:
Latest commit i removed this function and did the steps:
number -> sprite
sprite -> peep
check if peep is staff
I think this came out pretty clean now.